client/app/telecom/pack/xdsl/access/comfortExchange/pack-xdsl-access-comfort-exchange.controller.js

```diff
@@ -0,0 +1,82 @@
+import { COMFORT_EXCHANGE_TYPE_ERROR } from './pack-xdsl-access-comfort-exchange.constant';
+
+export default class XdslAccessComfortExchangeCtrl {
+  /* @ngInject */
+  constructor($q, $translate, OvhApiXdsl, TucToast, TucToastError) {
```

Comment:
This makes it easier to add new dependencies :)
```suggestion
  constructor(
    $q,
    $translate,
    OvhApiXdsl,
    TucToast,
    TucToastError,
  ) {
```